code/src/Submit.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Submit = ({ onClick }) => {
```

Comment:
This Submit component is the same like Start component. You can combine both of them into one button component, that will also have a title like parameter

Reply:
Thank you so much for this comment Sandra! I replaced my 3 buttons with one unique button 🥳